cmd/cli/serve/serve.go

```diff
@@ -173,6 +178,11 @@ func serve(cmd *cobra.Command) error {
 		return err
 	}
 
+	startWebUI, err := config.Get[bool](types.NodeWebUI)
+	if err != nil {
+		return err
+	}
```

Comment:
This is okay for now, but in future we'll probably want the web ui to be available on a configured port instead of the default port (80).  So it might be worth writing a ticket to capture this need in a future change?

Reply:
Agreed, will make a ticket for this

Reply:
https://github.com/bacalhau-project/expanso-planning/issues/133